src/definitions/modules/slider.js

```diff
@@ -1183,6 +1188,7 @@ $.fn.slider.settings = {
     reversed : 'reversed',
     disabled : 'disabled',
     labeled  : 'labeled',
+    noticks  : 'noticks',
```

Comment:
Maybe we should change this to `ticked` so it sounds more semantic. `ui ticked slider` sounds better then `ui noticks slider` and maybe `showLabelTicks` should also check if the class is present.

Reply:
Done.